.github/workflows/build.yml

```diff
@@ -207,22 +233,11 @@ jobs:
   push:
     name: Push Images
     runs-on: ubuntu-latest
-    needs: ["build", "test", "lint", "acceptance-test", "cypress"]
-    outputs:
-      branch: ${{ steps.set-outputs.outputs.branch }}
-      tag: ${{ steps.bump_version.outputs.tag }}
+    needs: [identify, "build", "test", "lint", "acceptance-test", "cypress"]
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: "0"
-      - name: Extract branch name
-        run: |
-          if [ "${{ github.head_ref }}" == "" ]; then
```

Comment:
Can we keep this format for identitying the branch name as it's compatible with workflow dispatch and the other isn't